src/components/studies/scheduler/ConfigureBurstTab.tsx

```diff
@@ -216,10 +325,31 @@ const ConfigureBurstTab: FunctionComponent<ConfigureBurstTabProps> = ({
                 bursts
               </FormControl>
             </FormGroup>
-            <SaveButton style={{margin: '0 auto 16px auto'}} />
+            <SaveButton />
           </div>
         </div>
       )}
+      {displayBurstInfoText && (
+        <Box className={classes.burstsInfoText}>
+          <Box className={classes.row}>
+            <img className={classes.calendarIcon} src={CalendarIcon}></img>
+            <Box
```

Comment:
can we use a class? Don't really need to specify the lineHeight. is fontSize 14 default?